### ISSUE TITLE
Please change filename in tutorial

### ISSUE THREAD

**Issue by ljcohen** [dib-lab/kevlar#163]
In the [tutorial](http://kevlar.readthedocs.io/en/stable/quick-start.html), the filename `refr.gz` does not exist. This command worked:

```
kevlar simplex \
    --case proband.fq.gz --case-min 6 \
    --control mother.fq.gz --control father.fq.gz --ctrl-max 0 \
    --novel-memory 5M --novel-fpr 0.6 --threads 4 \
    --filter-memory 1M --filter-fpr 0.005 \
    --mask-files refr.fq.gz --mask-memory 5M \
    --ksize 25 --out variant-calls.vcf refr.fq.gz
```

**Comment by standage** [comment]
Thanks for pointing this out @ljcohen!

**Comment by standage** [comment]
Fixed in 9f6bec06d4fdff0d0816eb57ca0527571ee5496c.
